src/CoamClient.js

```diff
@@ -95,6 +95,10 @@ class CoamClient {
         }
     }
 
+    buildGroupUrlFromId(groupId) {
```

Comment:
Can we also have a helper method for this so it kind of follows the general idea of the package (it allows you to work with or without explicitly creating the client)